agent/agent-api-public-deprecated.yaml

```diff
@@ -1,6 +1,6 @@
 swagger: '2.0'
 info:
-  version: '1.55.0'
+  version: '1.55.3'
```

Comment:
change this to 1.55.4

Reply:
is that right for the agent?
should it be 2.55.13, isn't it?